.circleci/config.yml

```diff
@@ -126,6 +137,35 @@ jobs:
           command: |
             poetry publish --username "${PYPI_USERNAME}" --password "${PYPI_PASSWORD}" --no-interaction
 
+  trigger-gitlab-pipeline:
+    docker:
+      - image: cimg/base:stable
+    steps:
+      - attach_workspace:
+          at: .
+      - run:
+          name: Configure the SSH deploy private key
+          command: |
+            mkdir -p ~/.ssh
+            echo "$DEPLOY_PRIVATE_KEY" > ~/.ssh/id_ed25519
+            chmod 600 ~/.ssh/id_ed25519
+            ssh-keyscan -t rsa gitlab.com >> ~/.ssh/known_hosts
+      - run:
+          name: Configure Git
+          command: |
+            git config --global user.email "root@data.gouv.fr"
+            git config --global user.name "datagouv"
+      - run:
+          name: Clone scaffold repository
+          command: |
+            git clone --quiet --depth 1 $SCAFFOLD_REPO_SSH_URL scaffold
+      - run:
+          name: Trigger Gitlab CI/CD pipeline for Hydra to deploy to dev environment
+          command: |
+            RELEASE_VERSION=$(cat version.txt)
+            cd scaffold
+            ./scripts/gitlab-ci-pipeline.sh hydra $RELEASE_VERSION dev ""
```

Comment:
Maybe add an explicit comment about the expected params? I can't easily know what `""` stands for.